singolo/index.html

```diff
@@ -22,12 +22,12 @@ <h1>singolo<span>*</span></h1>
         <nav class="header__navigation">
           <ul class="navigation">
             <li><a class="navigation__link" href="#header">home</a></li>
-            <li class="t">.</li>
+            <li class="dot">.</li>
```

Comment:
Do you think that the best idea to use an extra element in navigation for the separator element?

Reply:
This is not the best ,there are ways and better, but I decided to implement it this way.